bodhi/client/__init__.py

```diff
@@ -80,7 +80,8 @@ def new(user, password, **kwargs):
         except bindings.BodhiClientException as e:
             click.echo(str(e))
         except Exception as e:
-            traceback.print_exc()
+            sys.tracebacklimit = 0
+	    print("Wrong password, Try again")
```

Comment:
Let's use the click library for I/O instead of print statements, as it makes testing easier.